llvm/lib/Target/AMDGPU/SIISelLowering.cpp

```diff
@@ -16312,6 +16325,16 @@ bool SITargetLowering::checkForPhysRegDependency(
 }
 
 void SITargetLowering::emitExpandAtomicRMW(AtomicRMWInst *AI) const {
+  if (AI->getOperation() == AtomicRMWInst::Sub ||
```

Comment:
tiny nit: put `AI->getOperation()` in a variable maybe? it's also used below in older code